include/functions.h

```diff
@@ -9,24 +9,37 @@ void osCleanupThread(void);
 
 void clone_model(u16 srcModelID, u16 newModelID);
 void update_collider_transform(s16 colliderID);
-s32 get_variable(script_context* script, s32 pos);
-f32 get_float_variable(script_context* script, s32 pos);
-f32 set_float_variable(script_context* script, s32 dest, f32 value);
+s32 get_variable(script_context* script, s32 var);
+s32 set_variable(script_context* script, s32 var, s32 value);
+f32 get_float_variable(script_context* script, s32 var);
+f32 set_float_variable(script_context* script, s32 var, f32 value);
 void set_script_timescale(script_context* script, f32 timescale);
 f32 sin_deg(f32 x);
 f32 cos_deg(f32 x);
+f32 atan2(f32 startX, f32 startZ, f32 endX, f32 endZ);
+f32 clamp_angle(f32 theta);
 s32 func_80055448(s32);
 s32 func_80055464(s32, s32);
 s32 func_800554A4(s32, s32);
 s32 func_800554E8(s32, s32);
 s32 func_800555E4(s32);
 void func_80137D88(s32, f32);
+void func_80137DA4(s32, f32);
 void func_80137E10(s32, u8, u8, u8);
+void func_80137E4C(s32, s32, s32, s32);
 s32 rand_int(s32);
 void sort_items(void);
 s32 is_ability_active(s32 arg0);
 f32 update_lerp(EASING easing, f32 start, f32 end, s32 elapsed, s32 duration);
 
+npc* get_npc_safe(NPC npcID);
+npc* get_npc_unsafe(NPC npcID);
+npc* resolve_npc(script_context* script, NPC npcID);
```

Comment:
It might be worth declaring some kind of union for `resolve_npc`'s second arg - e.g. `union { npc* ptr; NPC special; u8 id }`. AFAIK the `get_npc` don't check for `NPC special` and should just take a `u8` id. We might even want to rename them - and suggest the same to Clover - to `get_npc_by_id`.

Reply:
Update: changing the `get_npc_` signatures to take a `u8` rather than an `s32` doesn't match.